config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+type Configuration struct {
+	Files []FileConfiguration
+}
+
+func (c *Configuration) GetFiles() []FileConfiguration {
+	return c.Files
+}
+
+type FileConfiguration struct {
+	InputFile   string
+	OutputFile  string
+	ModifyValue time.Duration
+}
+
+func GetConfiguration(path string) (*Configuration, error) {
```

Comment:
```suggestion
func FromPath(path string) (*Configuration, error) {
```
then it would read as `config.FromPath("./somepath/config.yml")`

Reply:
Go teaches us to use pkg names as prefixes to describe our methods.. since it's pkg config, ofc one can guess we're going to do something about config
exactly what? the func name can describe that

Reply:
Understood. Yes, it is much more descriptive. Even a non-tech can easily understand what is going on.